src/test/kotlin/com/terraformation/backend/accelerator/SpeciesSubmissionUpdaterTest.kt

```diff
@@ -0,0 +1,149 @@
+                      projectId = projectId,
+                      speciesId = speciesId,
+                      submissionStatus = SubmissionStatus.NotSubmitted,
+                  )))
+
+      val userId = currentUser().userId
+      val now = clock.instant
+
+      val actual = submissionsDao.fetchOneById(submissionId)
+      val expected =
+          SubmissionsRow(
+              createdBy = userId,
+              createdTime = now,
+              deliverableId = deliverableId,
+              feedback = "So far so good",
+              id = submissionId,
+              internalComment = "Internal comment",
+              modifiedBy = userId,
+              modifiedTime = now,
+              projectId = projectId,
+              submissionStatusId = SubmissionStatus.NotSubmitted)
+
+      assertEquals(expected, actual, "The submission's status was not updated to 'Not Submitted'")
+    }
+
+    @Test
+    fun `does nothing if the submission status is not 'Approved'`() {
+      val cohortId = insertCohort()
+      val participantId = insertParticipant(cohortId = cohortId)
+      val projectId = insertProject(participantId = participantId)
+      val speciesId = insertSpecies()
+      val moduleId = insertModule()
+      val deliverableId =
+          insertDeliverable(moduleId = moduleId, deliverableTypeId = DeliverableType.Species)
+      val submissionId =
+          insertSubmission(
+              deliverableId = deliverableId,
+              feedback = "So far so good",
+              internalComment = "Internal comment",
+              projectId = projectId,
+              submissionStatus = SubmissionStatus.InReview)
+
+      val participantProjectSpeciesId =
+          insertParticipantProjectSpecies(projectId = projectId, speciesId = speciesId)
+
+      updater.on(
+          ParticipantProjectSpeciesAddedEvent(
+              deliverableId = deliverableId,
+              participantProjectSpecies =
+                  ExistingParticipantProjectSpeciesModel(
+                      id = participantProjectSpeciesId,
+                      projectId = projectId,
+                      speciesId = speciesId,
+                      submissionStatus = SubmissionStatus.NotSubmitted,
+                  )))
+
+      val userId = currentUser().userId
+      val now = clock.instant
+
+      val actual = submissionsDao.fetchOneById(submissionId)
+      val expected =
+          SubmissionsRow(
+              createdBy = userId,
+              createdTime = now,
+              deliverableId = deliverableId,
+              feedback = "So far so good",
+              id = submissionId,
+              internalComment = "Internal comment",
+              modifiedBy = userId,
+              modifiedTime = now,
+              projectId = projectId,
+              submissionStatusId = SubmissionStatus.InReview)
+
+      assertEquals(expected, actual, "The submission's status was not updated to 'Not Submitted'")
```

Comment:
The assertion message here is the same as on the other test case but doesn't seem correct in this context.

Reply:
I simplified the test, hopefully it makes more sense now